dependencies.yaml

```diff
@@ -137,11 +161,11 @@ dependencies:
             - matrix:
                 cuda: "12.*"
               packages:
-                - librmm-cu12==24.8.*
+                - librmm-cu12==24.8.*,>=0.0.0a0
             - matrix:
                 cuda: "11.*"
               packages:
-                - librmm-cu11==24.8.*
+                - librmm-cu11==24.8.*,>=0.0.0a0
             - matrix:
               packages:
                 - librmm==24.8.*
```

Comment:
Needs an alpha spec too. Not critical for correctness here, but I think https://github.com/rapidsai/pre-commit-hooks/pull/28 will enforce this now.